functions.json

```diff
@@ -237,6 +237,18 @@
             "write_timeout": "60s"
         }
     },
+    {
+        "title": "nvidia-smi",
+        "name": "nvidia-smi",
+        "description": "Print NVIDIA System Management Interface (nvidia-smi) info",
```

Comment:
`Print output from nvidia-smi, requires a custom Profile with a request for an Nvidia GPU.`

Reply:
The description has been updated.